kubectl-rook-ceph.sh

```diff
@@ -230,7 +230,7 @@ function path_cm_rook_ceph_operator_config() {
 # 'kubectl rook-ceph mons' commands
 ####################################################################################################
 
-function run_mons_command () {
+function run_mons_command() {
```

Comment:
The changes in this script are just all from the automatic editor, right?

Reply:
yes.